src/queries.rs

```diff
@@ -756,7 +758,13 @@ impl<'a> VintedWrapper<'a> {
 
         match json.status() {
             StatusCode::OK => {
-                let items: AdvancedItems = json.json().await?;
+                let response_text = json.text().await?;
```

Comment:
I do not understand why this recoverable error you converted to an unrecoverable error.

What's the problem of having just `?` operator ?

Reply:
Because i need to previously store the response of the text in order to display it in case of error.